### ISSUE TITLE
Would it be possible to create a feed of all the tweets liked by an user?

### ISSUE THREAD

**Issue by xem** [ciderpunx/twitrssme#32]
Thanks!

**Comment by ciderpunx** [comment]
I don't think so, but if you can point me to a page in the public website that shows this, I can look into it.

**Comment by xem** [comment]
Here's an example of a page with all the likes of an user:

https://twitter.com/billgates/likes

(replace billgates with any username)
